sql/moz-fx-cjms-nonprod-9a36/cjms_bigquery/subscriptions_v1/query.sql

```diff
@@ -72,12 +146,30 @@ SELECT
   1 AS quantity,
   nonprod_stripe_subscriptions.plan_id, -- sku
   nonprod_stripe_subscriptions.plan_currency,
-  nonprod_stripe_subscriptions.plan_amount,
+  (
+    nonprod_stripe_subscriptions.plan_amount - COALESCE(amount_discounts.amount_off, 0) - COALESCE(
+      percent_discounts.amount_off,
+      0
+    )
+  ) AS plan_amount,
   nonprod_stripe_subscriptions.country,
   attributed_subs.flow_id,
+  promotion_codes.promotion_codes,
 FROM
   attributed_subs
 JOIN
   mozdata.subscription_platform.nonprod_stripe_subscriptions
 USING
   (subscription_id)
+LEFT JOIN
+  promotion_codes
+USING
+  (subscription_id)
+LEFT JOIN
+  amount_discounts
+USING
+  (subscription_id)
```

Comment:
As it stands, if there are multi-product subscriptions then any flat-rate discount will get applied to each product in full.  I considered adding logic to prorate flat-rate discounts in that scenario, but I ultimately decided not to since multi-product subscriptions aren't being used in production yet (ETA is July) and other changes will likely be needed here to support multi-product subscriptions.